pkg/controller/podautoscaler/horizontal.go

```diff
@@ -276,7 +276,7 @@ func (a *HorizontalController) processNextWorkItem(ctx context.Context) bool {
 
 	deleted, err := a.reconcileKey(ctx, key)
 	if err != nil {
-		utilruntime.HandleError(err)
+		utilruntime.HandleErrorWithContext(ctx, err, "")
```

Comment:
Please don't leave the message string empty. It should describe the context of the error and follow the conventions for log messages (start with capital letter, no punctuation at the end).